Dockerfile

```diff
@@ -1,10 +1,12 @@
-FROM quay.io/thoth-station/s2i-thoth-ubi8-py36
+FROM quay.io/thoth-station/s2i-thoth-f32-py38
 
-USER 0
+ENV LANG=en_US.UTF-8
+WORKDIR /home/si-cloc
+USER root
+RUN dnf update --setopt=tsflags=nodocs -y && \
+  dnf install --setopt=tsflags=nodocs -y cloc
 
-RUN dnf install cloc -y
-RUN pip install --upgrade pip -y
-RUN pip install pipenv -y
-RUN pipenv shell
+COPY ./* ${WORKDIR}
 
+RUN micropipenv install --dev
```

Comment:
This should be done by s2i python assemble logic. Hence the `COPY` instruction should not be needed as neither.

Reply:
This is a Dockerfile based build process. 
I'm missing out when would the s2i build process be used, since its docker file based.

Reply:
The produced artifact is based on s2i-thoth so it's s2i enabled (see `FROM` instruction). The only thing (only layer) that should be added to the container image is `dnf install --setopt=tsflags=nodocs -y cloc` so cloc is present in the image (and no dnf update please).

As the image is s2i-thoth based it will act as a normaln s2i Python application builder.

Reply:
In other words - the produced image by this Dockerfile is a builder image, not the application image.

Reply:
Got it, if we use the image as a builder image instead of an application image. :+1: